packages/@core.next/useEmitterMutation.ts

```diff
@@ -0,0 +1,47 @@
+import {
+    useMutation,
+} from '@apollo/client'
+import { get } from 'lodash'
+import { OperationVariables } from '@apollo/client/core'
+import { DocumentNode } from 'graphql'
+import { TypedDocumentNode } from '@graphql-typed-document-node/core'
+import { MutationFunctionOptions, MutationHookOptions, MutationTuple } from '@apollo/client/react/types/types'
+import { FetchResult } from '@apollo/client/link/core'
+import * as EventEmitter from 'eventemitter3'
```

Comment:
no need to add to package.json?

Reply:
Seems like it's already in global package.json
Should I add it here too manually?

Reply:
Added in core package.json explicitly